Libraries/Components/TextInput/TextInput.js

```diff
@@ -673,6 +690,12 @@ const TextInput = React.createClass({
     }
   },
 
+  _onContentSizeChange(event: Event) {
+    if (this.props.onContentSizeChange) {
+      this.props.onContentSizeChange(event.nativeEvent.contentSize);
```

Comment:
With destructuring assignment the syntax overhead of unpacking an object vs. getting just the relevant value is only `{ }` so I think we should go with the more future-proof approach and use the object.